### ISSUE TITLE
[BUG] latest release (v0.3.0) is failing in GitHub Actions

### ISSUE THREAD

**Issue by JuanJTorres11** [stackrox/kube-linter#335]
**System info:**
 - OS:Linux

**Describe the bug**
Since the release of the latest version, the GitHub Action stackrox/kube-linter-action@v1.0.4 is failing, checking the output of the GitHub Action I could see that it searches for the latest release and giving that it stop working around the same time as the launch of the latest release, I believe there is an error with it.

**To Reproduce**
I have the following step in a job of a workflow in GitHub Action 

```
  - name: Scan YAMLs
         uses: stackrox/kube-linter-action@v1.0.4
         id: kube-linter-action-scan
         with:
           directory: yamls
```
It was working perfectly until the latest release was launched some hours ago.

**Sample YAML input**
If applicable, sample YAML input which reproduces the issue.

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/43612178/168170841-3cb8eba3-dab0-456c-b08f-8b7f9b9e006f.png)

**Additional context**

**Comment by miloslaptosevic** [comment]
This looks like an issue with renaming of assets in release. Kube linter github action uses this grep `grep --fixed-strings "kube-linter-${OS}.tar.gz")` to match asset in release but in new release (v0.3.0) assets have been renamed from `kube-linter-${OS}.tar.gz` and `kube-linter-${OS}.zip` to just `kube-linter-${OS}`.

**Comment by svenjr** [comment]
As a workaround, you can pin the correct version until this is fixed. So that would look like:

```yaml
  - name: Scan YAMLs
         uses: stackrox/kube-linter-action@v1.0.4
         id: kube-linter-action-scan
         with:
           directory: yamls
           version: "0.2.6" # Avoid v0.3.0 which is broken
```
